### PR TITLE
First section

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/css/first-section.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.first-section-title {
+  text-align: center;
+  padding-top: 24px;
+  font-weight: var(--bold-font);
+}
+
+.first-section-container {
+  padding-top: 24px;
```
Comment (takrami) [inline assets/css/first-section.css]: Please use spacing variable for all spaces. See [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()). Will help you :D

File: index.html [hunk 2]
```diff
@@ -21,9 +21,22 @@
     </header>
     <section class="first-section">
       <div class="site-wrapper">
+        <div class="first-section-title">Web & Application Development</div>
```
Comment (takrami) [inline index.html]: You can use h tag here.

Comment (hedita) [reply index.html]: Ok.

File: assets/css/first-section.css (hunk 1 above)
Comment (hedita) [reply assets/css/first-section.css]: Done.

File: assets/css/first-section.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  font-weight: var(--bold-font);
+}
+
+.first-section-container {
+  padding-top: calc(var(--spacing) * 4);
+  padding-bottom: calc(var(--spacing) * 4);
+  display: grid;
+  justify-content: center;
+}
+
```
Comment (takrami) [inline assets/css/first-section.css:17]: You need line-height for the first-section-item.

**Review by takrami — APPROVED** [review]
Great job! :1st_place_medal: I can see you need line hight in most of the places. It's better to add it in general css.

File: index.html [hunk 4]
```diff
@@ -21,9 +21,22 @@
     </header>
     <section class="first-section">
       <div class="site-wrapper">
+        <h2 class="first-section-title">Web & Application Development</h2>
         <div class="first-section-container">
-          <div class="first-section-item">1</div>
-          <div class="first-section-item">2</div>
+          <div class="first-section-item">
```
Comment (takrami) [inline index.html]: They are paragraph so can be p tag.

Comment (hedita) [reply index.html]: Done.

**Pull request merged**